packages/orbit-components/.storybook/preview.ts

```diff
@@ -29,6 +29,25 @@ const preview: Preview = {
   decorators: [orbitDecorator],
   parameters: {
     viewport: { viewports },
+    a11y: {
+      element: "#storybook-root",
+      config: {
+        rules: [
+          {
+            id: "aria-required-attr",
+            selector: "*:not(div.text-heading-title0[role='heading'])",
+          },
+          {
+            id: "color-contrast",
+            selector: "*:not(pre .attr-name.tag.token)",
+          },
+          {
+            id: "color-contrast-enhanced",
+            selector: "*:not(pre .attr-name.tag.token)",
+          },
```

Comment:
I believe it is safer to add a custom class to the [whole code snippet div](https://github.com/kiwicom/orbit/blob/e9327f465f6cb88cc6b3ebf456fc3dd0526a5058/packages/orbit-components/.storybook/orbitDecorator.tsx#L55) and target it, WDYT?